rotkehlchen/csv_exporter.py

```diff
@@ -103,6 +105,21 @@ def __init__(
         self.all_events: List[Dict[str, Any]] = []
         self.reset()
 
+        # get setting for prefered eth explorer
+        user_settings = self.database.get_settings()
+        try:
+            frontend_settings = json.loads(user_settings.frontend_settings)
+            if (
+                'explorers' in frontend_settings and
+                'ETH' in frontend_settings['explorers'] and
+                'transaction' in frontend_settings['explorers']['ETH']
+            ):
+                self.eth_explorer = frontend_settings['explorers']['ETH']['transaction']
+            else:
+                self.eth_explorer = ETH_EXPLORER
+        except json.decoder.JSONDecodeError:
```

Comment:
you need to catch a key error here too

Reply:
I thought about a keyerror but for how is the condition of that `if` it can never happen. Maybe I should have removed that if completely and just catch the error